core/src/main/kotlin/com/justai/jaicf/activator/strict/ButtonActivator.kt

```diff
@@ -0,0 +1,29 @@
+package com.justai.jaicf.activator.strict
+
+import com.justai.jaicf.activator.Activator
+import com.justai.jaicf.activator.ActivatorFactory
+import com.justai.jaicf.activator.selection.ActivationSelector
+import com.justai.jaicf.api.BotRequest
+import com.justai.jaicf.api.hasQuery
+import com.justai.jaicf.context.BotContext
+import com.justai.jaicf.context.StrictActivatorContext
+import com.justai.jaicf.model.activation.Activation
+import com.justai.jaicf.model.scenario.ScenarioModel
+
+class ButtonActivator : Activator {
+
+    override val name = "buttonActivator"
+
+    override fun canHandle(request: BotRequest): Boolean = request.hasQuery()
+
+    override fun activate(botContext: BotContext, request: BotRequest, selector: ActivationSelector): Activation? {
+        val req = request.input.toLowerCase()
```

Comment:
Why `toLowerCase`? I think that button press preserves the text case and case mismatching shows that the text was typed manually

Reply:
As I remember, some channels may convert button input to upper or lower case